src/ServicePulse.Host/vue/.prettierrc.json

```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+  "printWidth": 200
```

Comment:
I think if you add
```
"tabWidth": 4,
```
it will fix the reformatting everywhere

Reply:
this will make it worse as far as I can see, since the majority of the tabbing in the solution is 2 spaces